Dockerfile

```diff
@@ -42,4 +40,4 @@ EXPOSE 5030
 
 ENTRYPOINT ["/docker-entrypoint.sh"]
 
-CMD [ "granian", "--interface", "wsgi", "--host", "0.0.0.0", "--port", "5030", "apiserver:app" ]
+CMD [ "granian", "--interface", "wsgi", "--host", "0.0.0.0", "apiserver:app" ]
```

Comment:
Where do we configure the port now?

Reply:
Directly in ops manifests, see https://github.com/getsentry/ops/blob/master/k8s/services/release-registry/deployment.yaml#L20